odoo/models.py

```diff
@@ -489,12 +491,12 @@ class BaseModel(metaclass=MetaModel):
     .. seealso:: :class:`TransientModel`
     """
 
-    _name = None                #: the model name (in dot-notation, module namespace)
-    _description = None         #: the model's informal name
+    _name: Union[str, None] = None                #: the model name (in dot-notation, module namespace)
+    _description: Union[str, None] = None         #: the model's informal name
     _module = None              #: the model's module (in the Odoo sense)
     _custom = False             #: should be True for custom models only
 
-    _inherit = ()
+    _inherit: Union[str, list[str]] = ()
```

Comment:
This one is normalized by the metaclass `MetaModel`, so it is always a `list` or `tuple` of `str`.  But I don't know how confusing it may be for the type checker when you have something like
```py
class Foo(Model):
    _inherit = 'foo'

Foo._inherit    # returns ['foo']
```